packages/subapp-web/package.json

```diff
@@ -51,14 +51,15 @@
     "@babel/preset-react": "^7.16.7",
     "@babel/register": "^7.7.7",
     "@xarc/app": "^11.0.0",
+    "@xarc/run": "^1.1.1",
```

Comment:
Hi @ashuverma Was working on from your fork and was wondering does this package need "@xarc/run" ?

Reply:
It was getting used but not defined in `package.json`. rush+pnpm doesn't allow phantom dependencies.

Reply:
ok sure 👍 I will double check it - I was not getting error in this one but had to add on another package though